observability/otel/otel-collector/otel-collector-extensions.adoc

```diff
@@ -142,10 +142,10 @@ include::snippets/technology-preview.adoc[]
     exporters:
       otlp:
         sending_queue:
-          storage: file_storage/all_settings
+          storage: file_storage/all_settings # <7>
 
     service:
-      extensions: [file_storage/all_settings]
+      extensions: [file_storage/all_settings] # <8>
```

Comment:
The brackets are intended in this entry and not the other entry?

Reply:
@frzifus, ^ this is a question for you.

Reply:
Yes, that entry is a list where multiple extensions can be registered.